metricbeat/module/kubernetes/controllermanager/_meta/fields.yml

```diff
@@ -0,0 +1,128 @@
+- name: controllermanager
+  type: group
+  description: >
+    Controller manager metrics
+  release: ga
+  fields:
+    - name: handler
+      type: keyword
+      description: >
+        Request handler
+    - name: code
+      type: keyword
+      description: >
+        HTTP code
+    - name: method
+      type: keyword
+      description: >
+        HTTP method
+    - name: host
+      type: keyword
+      description: >
+        Request host
+    - name: name
+      type: keyword
+      description: >
+        Name for the resource
+    - name: zone
+      type: keyword
+      description: >
+        Infrastructure zone
+    - name: process
+      type: group
+      fields:
+        - name: cpu.sec
+          type: double
+          description: CPU seconds
+        - name: memory.resident.bytes
+          type: long
+          format: bytes
+          description: Bytes in resident memory
+        - name: memory.virtual.bytes
+          type: long
+          format: bytes
+          description: Bytes in virtual memory
+        - name: fds.open.count
+          type: long
+          description: Number of open file descriptors
+        - name: started.sec
+          type: double
+          description: Seconds since the process started
+    - name: http
+      type: group
+      fields:
+        - name: request.duration.us.percentile.*
+          type: object
+          object_type: double
+          description: Request duration microseconds percentiles
+        - name: request.duration.us.sum
+          type: double
+          description: Request duration microseconds cumulative sum
+        - name: request.duration.us.count
+          type: long
+          description: Request count for duration
+        - name: request.size.bytes.percentile.*
+          type: object
+          object_type: long
+          description: Request size percentiles
+        - name: request.size.bytes.sum
+          type: long
+          format: bytes
+          description: Request size cumulative sum
+        - name: request.size.bytes.count
```

Comment:
This metric is really called `request.size.bytes` according to the mapping in the code. I wonder why this is not being checked by the fields validator.

Reply:
This is `http.request.size.bytes.count`.
Can you elaborate the comment?

Reply:
Oh sorry, In the code I saw it as `http.request.size.bytes`, without the `count`:
```
 			"http_response_size_bytes":                    prometheus.Metric("http.response.size.bytes"), 
``` 
But I guess this is an histogram :man_facepalming: